controllers/dashboard.go

```diff
@@ -119,7 +110,18 @@ func (c *DashboardController) FileSearch() {
 	flash := web.NewFlash()
 	redirectPath := "/dashboard"
 
-	flash.Success("Scraping all keywords :)")
+	file, fileHeader, err := c.GetFile("file")
+	if err != nil {
+		flash.Error("The specified file could not be uploaded :(")
+	} else {
+		fileForm := form.FileSearchForm{File: file, FileHeader: fileHeader, User: c.CurrentUser}
+		err = fileForm.Save()
+		if err != nil {
+			flash.Error(err.Error())
+		} else {
+			flash.Success("Scraping all keywords :)")
```

Comment:
```suggestion
			flash.Success("Scraping all keywords:)")
```

Reply:
@junan it's emoji :P, just make it more fun in message